pkg/shed/db.go

```diff
@@ -74,83 +87,359 @@ func NewDB(path string, logger logging.Logger) (db *DB, err error) {
 		}
 	}
 
-	// Create a quit channel for the periodic metrics collector and run it
-	db.quit = make(chan struct{})
-
+	db.metrics.DBOpenCount.Inc()
```

Comment:
not sure if this metric is all that useful. this should always be 1, at least for the time being...